specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9872 @@
+            "modelAsString": true
+          }
+        }
+      },
+      "discriminator": "kind"
+    },
+    "CredentialList": {
+      "description": "List of credentials.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of credentials.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Credential"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of results, if any remaining results exist.",
+          "type": "string"
+        },
+        "count": {
+          "description": "The number of credentials.",
+          "format": "int64",
+          "type": "integer"
+        }
+      }
+    },
+    "Notification": {
+      "description": "The notification model.",
+      "type": "object",
+      "properties": {
+        "message": {
+          "type": "string",
+          "description": "Notification message."
+        },
+        "code": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Notification code."
+        }
+      }
+    },
+    "ScanDiagnostics": {
+      "description": "The scan diagnostics.",
+      "type": "object",
+      "properties": {
+        "notifications": {
+          "type": "array",
+          "description": "Notifications.",
+          "items": {
+            "$ref": "#/definitions/Notification"
+          }
+        },
+        "exceptionCountMap": {
+          "type": "object",
+          "description": "Exception count map.",
+          "additionalProperties": {
+            "format": "int32",
+            "type": "integer"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "DataSourceType": {
+      "description": "The data source type.",
+      "enum": [
+        "None",
+        "Fabric",
+        "AzureSubscription",
+        "AzureResourceGroup",
+        "AzureSynapseWorkspace",
+        "AzureSynapse",
+        "AdlsGen1",
+        "AdlsGen2",
+        "AmazonAccount",
+        "AmazonS3",
+        "AmazonSql",
+        "AzureCosmosDb",
+        "AzureDataExplorer",
+        "AzureFileService",
+        "AzureSqlDatabase",
+        "ArcEnabledSqlServer",
+        "AmazonPostgreSql",
+        "AzurePostgreSql",
+        "Databricks",
+        "SqlServerDatabase",
+        "AzureSqlDatabaseManagedInstance",
+        "AzureSqlDataWarehouse",
+        "AzureMySql",
+        "Hdfs",
+        "TableauServer",
+        "AzureStorage",
+        "Teradata",
+        "Oracle",
+        "PostgreSql",
+        "AmazonRedShift",
+        "DatabricksHms",
+        "SapS4Hana",
+        "SapEcc",
+        "Snowflake",
+        "PowerBI",
+        "Trident",
+        "Dataverse",
+        "DatabricksUnityCatalog"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "DataSourceType",
+        "modelAsString": true
+      }
+    },
+    "ScanResult": {
```

Comment:
Is it used anywhere?

Reply:
Just removed it. This is an old version of scanresultwithingestion

Reply:
Maybe you also want to check the name of `scanresultwithingestion`. Should it be `scanresult` instead?

Just a note.

Reply:
Yes, previously, we are using scanResult, and the scanResultwithIngestion is the newer version of scanResult, we would like to keep these 2 versions separately to avoid changing most of our legacy code that supports older versions. And you are right, for API versions from GA and later on, we are only keeping scanResultWithIngestion as the only scan result object.